llpc/context/llpcCompiler.cpp

```diff
@@ -778,7 +778,7 @@ Result Compiler::BuildShaderModule(const ShaderModuleBuildInfo *shaderInfo, Shad
 // @param shaderInfo : Shader info of this pipeline
 // @param [out] pipelineElf : Output Elf package
```

Comment:
You did not add a comment for the new parameter.  Be sure to call out that you expect it to already be initialized.